data/explorations/welcome/welcome.yaml

```diff
@@ -231,31 +231,31 @@ states:
       - outcome:
           dest: Numeric input
           feedback:
-            content_id: feedback_4
-            html: <p>That's too high! Try guessing lower.</p>
+            content_id: feedback_5
+            html: <p>That's not large enough. Try guessing higher?</p>
           labelled_as_correct: false
           missing_prerequisite_skill_id: null
           param_changes: []
           refresher_exploration_id: null
         rule_specs:
         - inputs:
-            x: 121.0
-          rule_type: IsGreaterThan
+            x: 131.0
```

Comment:
These changes seem wrong to me. What was wrong with how it was before? I would still like the demo explorations to make sense.